incubator/model-json/src/main/java/io/aklivity/zilla/runtime/model/json/internal/config/JsonModelConfigAdapter.java

```diff
@@ -86,7 +84,7 @@ public ModelConfig adaptFromJson(
         for (String catalogName: catalogsJson.keySet())
         {
             JsonArray schemasJson = catalogsJson.getJsonArray(catalogName);
-            ObjectHashSet<SchemaConfig> schemas = new ObjectHashSet<>();
+            List<SchemaConfig> schemas = new LinkedList<>();
```

Comment:
Since we are using `List`, lets keep exposing it as a `List` in the config object instead of `Collection`.